en/docs/services/create-your-first-service.md

```diff
@@ -18,7 +18,7 @@ To create the Choreo service, follow the procedure below:
 
 4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.
 
-    ![Resource Configuration](assets/img/services/configure-api-trigger.png)
+    ![Resource Configuration](/assets/img/services/configure-api-trigger.png)
```

Comment:
```suggestion
    ![Resource Configuration](./assets/img/services/configure-api-trigger.png)
```

Shall we use `./` notation to make it the generated site portable.

Reply:
having a relative path to the image in the source file will make it difficult to move the source file within the structure. Hence used the image path from the root.